app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
```

Comment:
It's generally a good practice to avoid initializing variables that might not be used. In your case, if the `VaccineError` is raised in the first iteration, `masks_to_buy` will not be used. Consider moving this line inside the `try` block.